src/kajal/StringAndCharProgram/Assignment_30.java

```diff
@@ -0,0 +1,30 @@
+/*WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 */
+
+package kajal.StringAndCharProgram;
+
+public class Assignment_30 {
+
+	int returnSecondMax(int[] input) {
+		int max = input[0];
+		int secondMax = 0;
+		for (int index = 0; index < input.length; index++) {
+
+			if (max < input[index]) {
```

Comment:
this will fails for below input
int input[] = {10, 33, 43, 55, 97, 111, 99, 3};

to handle above one, need to write below code

Reply:
for (int index = 0; index < input.length; index++) {

            if (max < input[index]) {
                secondMax = max;
                max = input[index];
            } else if (input[index] > secondMax && max != input[index]) {
                secondMax = input[index];
            }
        }